pkg/services/ngalert/state/historian/loki_http.go

```diff
@@ -81,13 +90,13 @@ type Selector struct {
 	Value string
 }
 
-func newLokiClient(cfg LokiConfig, logger log.Logger) *httpLokiClient {
+func newLokiClient(cfg LokiConfig, req client.Requester, metrics *metrics.Historian, logger log.Logger) *httpLokiClient {
+	coll := instrument.NewHistogramCollector(metrics.WriteDuration)
```

Comment:
nit: I'm 50/50 on wether this belongs directly on the metrics package vs here. 

I don't mind overall but just mentioning it in case you considered (you probably did).

Reply:
I want to keep it here for the sake of this PR.
1. All the other metrics are in `metrics` and I don't want to be locally inconsistent even if its the eventual direction. Let's split them ALL out in the future.
2. If I move it in this PR, i'll have an absolute mess of merge conflicts on my hands when I rebase https://github.com/grafana/grafana/pull/63157

Reply:
Discussed offline, I misunderstood this comment. We're now wrapping the collector in `metrics`.